dotcom-rendering/src/web/components/CommentCount.stories.tsx

```diff
@@ -30,11 +49,8 @@ export const SpecialReportStory = () => {
 				isCommentable={true}
 				setIsExpanded={() => {}}
 				commentCount={306}
-				palette={decidePalette({
-					theme: ArticleSpecial.SpecialReport,
-					display: ArticleDisplay.Standard,
-					design: ArticleDesign.Standard,
-				})}
+				palette={decidePalette(specialReportFormat)}
+				format={specialReportFormat}
```

Comment:
I remember a while ago I was having this chat with oliver and he was also suggesting that we can only pass format instead of both format and palette and then create the palette within the component. 

@oliverlloyd Can this approach be done by gradually refactoring the existing components to only pass the format? OR as Jamie said this was avoided for performance reasons?

Reply:
Gradually refactoring is fine here. This is definitely a case where moving things around until the feel right is a sound approach.

Re performance. I don't think the generation of `palette` takes a hugely long time, I would guess we're in microseconds territory here, but at some point as this object grows we can certainly think about breaking the logic apart and exporting individual components 👍